src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return function(firstNumber) {
+    return function(secondNumber) {
+      counter++;
+
+      if (counter >= 3 && counter % 2 === 0) {
+        return 'Bzzz... Error!';
+      } else {
+        return firstNumber + secondNumber;
+      }
```

Comment:
`else` is redundant here.

```suggestion
      }
      
      return firstNumber + secondNumber;
```